backend/blueprints/spa_api/service_layers/replay/tag.py

```diff
@@ -128,33 +132,30 @@ def get_encoded_private_key(name: str, session=None) -> str:
     @staticmethod
     def encode_tag(tag_id: int, private_id: str) -> str:
         merged = str(tag_id + 1000) + ":" + private_id
-        return base64.b85encode(merged.encode(encoding="utf-8")).decode('utf-8')
+        return quote(base64.b85encode(merged.encode(encoding="utf-8")).decode('utf-8'))
```

Comment:
what's this

Reply:
This just makes it so it is url friendly for output instead of a bunch of weird characters

Reply:
could do with a comment imo.